velox/functions/prestosql/tests/ProbabilityTest.cpp

```diff
@@ -155,5 +155,63 @@ TEST_F(ProbabilityTest, binomialCDF) {
       binomialCDF(-2, 0.5, -1), "numberOfTrials must be greater than 0");
 }
 
+TEST_F(ProbabilityTest, invBinomialCDF) {
+  const auto invBinomialCDF = [&](std::optional<int64_t> numberOfTrials,
+                                  std::optional<double> successProbability,
+                                  std::optional<double> p) {
+    return evaluateOnce<int64_t>(
+        "inverse_binomial_cdf(c0, c1, c2)",
+        numberOfTrials,
+        successProbability,
+        p);
+  };
+
+  EXPECT_EQ(0, invBinomialCDF(5, 0.5, 0.03125));
```

Comment:
Nit : Some of the combinations here could be removed as they seem repetitive. e.g. There are 2 examples with successProbability and p (0.5, 0.5), and having so many scale digits also seems unnecessary since there is already an example with 4 scale digits.

Reply:
Fixed in most recent commit `f2a1493`